testsuite/MDAnalysisTests/utils/test_modelling.py

```diff
@@ -253,8 +251,8 @@ def test_merge_with_topology(self):
         # One of these bonds isn't in the merged Universe
         assert_(len(ag2[0].bonds) - 1 == len(u2.atoms[20].bonds))
 
-    def test_merge_with_topology_from_different_universes(self):
-        u3 = MDAnalysis.Merge(self.u.atoms[:110], self.u2.atoms)
+    def test_merge_with_topology_from_different_universes(self, u, u2):
+        u3 = MDAnalysis.Merge(u.atoms[:110], u2.atoms)
```

Comment:
This variable was named `u3` because the function was already using `self.u` and `self.u2`. Beside the fact that `u3` is a terrible name for a variable in general, there is no `u2` any more as one could expect from the name `u3`.

This variable cannot keep be called `u3`. I suggest renaming it `u_merge`. `u2` in the test above and the test bellow could be renamed `u_merge` as well. This name makes it clear that they point to the universe produced by the merge.